src/lib/detector/MACE/PhaseI/Detector/Description/TTC.c++

```diff
@@ -19,45 +19,46 @@ using namespace Mustard::PhysicalConstant;
 TTC::TTC() : // clang-format off
     DescriptionWithCacheBase{"TTC"}, // clang-format on
     // Geometry
-    fLength{this, 5.5_cm},
+    fLength{this, 5.2_cm},
     fWidth{this, 5_cm},
     fThickness{this, 0.5_cm},
-    fRadius{this, {8.5_cm, 8.5_cm, 8.5_cm, 8.5_cm, 8.5_cm, 8.5_cm, 8.5_cm, 8.5_cm, 8.5_cm}},
+    fRadius{this, {8.5_cm,8.5_cm,8.5_cm,8.5_cm,8.5_cm,8.5_cm,8.5_cm,8.5_cm,8.5_cm}},
```

Comment:
I believe this cannot pass all CI check since code is not fully formatted. Please merge from `main` branch and check CI status.